lib/flash_flow/deploy.rb

```diff
@@ -88,7 +88,10 @@ def merge_pull_requests
         if remote.nil?
           raise RuntimeError.new("No remote found for #{pull_request.head.repo.ssh_url}. Please run 'git remote add *your_remote_name* #{pull_request.head.repo.ssh_url}' and try again.")
         end
-        merge_or_rollback(remote, pull_request.head.ref, pull_request.number)
+
+        unless @github.has_label?(pull_request.number, @github.do_not_merge_label)
```

Comment:
I like the generalization of the `has_label?` method. But my point is really about the `do_not_merge_label` instance variable. It's never used in the class, the only time it's ever used it's just passed back in from deploy.rb. So you should either use it, or get rid of it from that class.

You could probably just do this if you want, and get rid of that variable from the class:
`unless @github.has_label?(pull_request.number, Config.configuration.do_not_merge_label)`